index.html

```diff
@@ -0,0 +1,228 @@
+            </p>
+          </article>
+        </section>
+      </section>
+      <section class="text-centered">
+        <section class="section-inner">
+          <article>
+            <h2>Gute Ideen für ein besseres Klima</h2>
+            <p>
+              Ihr wollt wissen, was ihr jetzt und in Zukunft tun könnt? Wir
+              haben die neuesten Trends und praktische Lösungen für euren
+              Alltag.
+            </p>
+            <section class="container-teaser-outer">
+              <section class="container-teaser-inner">
+                <figure>
+                  <img
+                    src="./assets/img/man-and-car-image-15.jpg"
+                    alt="Man und Auto"
+                  />
+                </figure>
+                <section class="container-teaser-inner-text">
+                  <h3>E-Mobilität ist kein Traum. Sondern Zukunftsmusik</h3>
+                  <p>
+                    Ob Wallbox, Ladekarte oder Stromtarif: Mit innovativen
+                    Lösungen rund ums E-Auto macht LichtBlick euch fit für die
+                    Mobilität von morgen. Und übbermorgen, und nächste Woche
+                    auch.
+                  </p>
+                  <p><a href="#" class="link-teaser">Zur E-Mobbilität</a></p>
+                </section>
+              </section>
+              <section class="container-teaser-inner">
+                <figure>
+                  <img
+                    src="./assets/img/klima-podcast-media.jpg"
+                    alt="Klimaneutralität"
+                  />
+                </figure>
+                <section class="container-teaser-inner-text">
+                  <h3>Klimaneutralität ist machbar. Und hörbar</h3>
+                  <p>
+                    Wie gestalten wir gemeinsam eine klimaneutrale Zukunft? Und
+                    was können wir alle jetzt schon im Alltag tun? In unserem
+                    Podcast erfahrt ihr's!
+                  </p>
+                  <p><a href="#" class="link-teaser">Zum Podcast</a></p>
+                </section>
+              </section>
+            </section>
+          </article>
+        </section>
+      </section>
+      <!-- ende 2er Block -->
+      <section class="bck-standard-green text-centered">
+        <section class="section-inner">
+          <article>
+            <h2>
+              LichtBlick erarbeitet 5 Klimaformen für Klimaneutralität bis 2035
+            </h2>
+            <p>
+              Die Wahl 2021 wird zur Klimawahl. Ihr Ausgang entscheidet, ob wir
+              die Klimaziele des Pariser Abkommens erreichen. Klar ist: Es
+              braucht große politische Veränderungen.
+            </p>
+            <p>
+              <a href="#" class="button">Zu den Reformen</a>
+            </p>
+          </article>
+        </section>
+      </section>
+    </main>
+    <footer class="footer-main">
+      <section class="bck-standard-dark">
```

Comment:
Benutzt du die `section` hier noch semantisch oder hast du einfach nur ein Block-Level Element für Styling-Gründe gebraucht? Dann wäre `div` die Wahl gewesen. Bitte nochmal prüfen

Reply:
würde sagen: ist schon eine semantische section, allerdings, weil es sonst keine section mehr hier gibt, kann es auch ein div sein. Habe es in div umgewandelt.